### PR TITLE
12.0 lcc members initial release

### PR DESCRIPTION
**Pull request by njeudy**
- Initial release of lokavaluto members modules

### CODE REVIEW

File: lcc_members/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+The development of this module has been financially supported by:
+
+* Lokavaluto (https://lokavaluto.fr)
+* Mycéliandre (https://myceliandre.fr)
+
+Maintainer
+----------
+
+This module is maintained by the LOKAVALUTO.
```
Comment (mazda91) [inline lcc_members/__manifest__.py]:
```suggestion
This module is maintained by LOKAVALUTO.
```

File: lcc_members/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+from odoo.tools.translate import _
+
+
+class res_partner(models.Model):
+    _inherit = "res.partner"
+
+    reasonschoosingmlc = fields.Text(
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
    reasons_choosing_lcc = fields.Text(
```

File: lcc_members/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        string=_("Why did I choos local monay"),
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
        string=_("Why did I choose local currency"),
```

File: lcc_members/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        readonly=False
+    )
+    phone_pro = fields.Char(
+        string=_("Professional phone"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    member_comment = fields.Text(
+        string=_("Commentaires membre"),
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
        string=_("Member comment"),
```

File: lcc_members/models/res_partner.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    member_type_id = fields.Many2one(
+        string=_("Type de membres"),
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
        string=_("Member's type"),
```

File: lcc_members/models/res_partner.py [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+        translate=False,
+        readonly=False
+    )
+    keywords = fields.Text(
+        string=_("keywords"),
+        required=False,
+        translate=True,
+        readonly=False
+    )
+    currencyexchangeoffice = fields.Boolean(
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
    is_currency_exchange_office = fields.Boolean(
```

File: lcc_members/models/res_partner.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+        translate=False,
+        readonly=False
+    )
+    twitter_url = fields.Char(
+        string=_("Twitter"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    openingtime = fields.Text(
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
    opening_time = fields.Text(
```

File: lcc_members/models/res_partner.py [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+        readonly=False
+    )
+    convention_specific_agreement = fields.Text(
+        string=_("Convention specific agreement"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    currency_exchange_office = fields.Boolean(
+        string=_("Comptoir de change"),
```
Comment (mazda91) [inline lcc_members/models/res_partner.py]:
```suggestion
        string=_("Currency exchange office"),
```

**Comment by njeudy** [comment]
thanks @mazda91 for the field will commit an other file because à need to apply it in code and associated 
views

**Comment by njeudy** [comment]
For view, all texte should be in english, and then translated in .po file.
Will commit this and update this merge request

File: lcc_members/models/res_partner.py (hunk 6 above)
Comment (njeudy) [reply lcc_members/models/res_partner.py]: @jcleonard2018 we need to remove this as asked in an other task right ?

**Comment by njeudy** [comment]
@mazda91 I push new revision to merde with all things you asked :)

Think I will merge this and go on views détails after (because laRoue can test and review and ask for some improvement.)

File: lcc_members/models/res_partner.py (hunk 6 above)
Comment (jcleonard2018) [reply lcc_members/models/res_partner.py]: Yes currency exchange office is managed using partner tags

**Pull request merged**